### PR TITLE
Update circle-mode.rst

### PR DESCRIPTION
**Pull request by timtuxworth**
I thought it would be better to refer to the parameter rather than 'hard coding' the timeout amount here.

### CODE REVIEW

File: plane/source/docs/circle-mode.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ CIRCLE Mode
 
 Circle mode is similar to :ref:`LOITER <loiter-mode>`, but doesn't attempt
 to hold position. This is primarily meant as a failsafe mode and is the
```
Comment (Hwurzburg) [inline plane/source/docs/circle-mode.rst:8]:
```suggestion
to hold position. This is primarily meant to be used as an initial failsafe mode to allow the vehicle to turn in order to hopefully regain radio control signal and pilot control.
```

File: plane/source/docs/circle-mode.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ CIRCLE Mode
...
-mode that the aircraft will enter by default for 20 seconds when a
+mode that the aircraft will enter by default for ``FS_LONG_TIMEOUT`` seconds when a
```
Comment (Hwurzburg) [inline plane/source/docs/circle-mode.rst:9]: remove all following.

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
this is incorrect....FS_SHORT_TIMEOUT is the time between radio loss and entering FS_SHORT_ACTN....read the https://ardupilot.org/plane/docs/apms-failsafe-function.html wiki

probably better just to remove any mention of the details and defaults....and just say what the mode does...failsafe options and details are explained on the failsafe wiki page

File: plane/source/docs/circle-mode.rst (hunk 2 above)
Comment (timtuxworth) [reply plane/source/docs/circle-mode.rst:9]: Would it make sense to include a reference/link to the Failsafe doc page (and specifically the section that refers to actions here: https://ardupilot.org/plane/docs/apms-failsafe-function.html#failsafe-parameters-and-their-meanings)
How can I add that link?

Comment (Hwurzburg) [reply plane/source/docs/circle-mode.rst:9]: See :ref:`failsafe documentation <apms-failsafe-function>` for more information.

Comment (Hwurzburg) [reply plane/source/docs/circle-mode.rst:9]: the backtick does not appear in markdown comments like these...just highlites the phrase

Comment (timtuxworth) [reply plane/source/docs/circle-mode.rst]: I don't know if I got it right - a backtick didn't show up in what you posted, so I just copied the text.

Comment (Hwurzburg) [reply plane/source/docs/circle-mode.rst]: fixed and squashed....merging

**Pull request merged**